Assets/Scripts/Clicker.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+
+public class Clicker : MonoBehaviour
+{
+  
+    public int power = 0;
+    public float powerPerSecond = 1.0f;
+    public TMP_Text powerText;
+
+    void Start()
```

Comment:
Unused method

Reply:
powerPerSecond is needed for OfflineProgress